lib/src/pubspec.dart

```diff
@@ -43,6 +48,8 @@ class Pubspec {
 
   Map<String, Dependency> get devDependencies => _inner.devDependencies;
 
+  Map get environment => _content['environment'] as Map;
```

Comment:
nit: `(_content['environment'] as Map) ?? {}` might be nicer in the future...

Reply:
Done